fireapp/lib/presentation/calendar/calendar_widget.dart

```diff
@@ -44,20 +35,346 @@ class _CalendarState extends FireAppState<CalendarView>
     implements ViewModelHolder<CalendarViewModel> {
   @override
   CalendarViewModel viewModel = GetIt.instance.get();
+  final List<Color> colorPalette = [
```

Comment:
Please define these in styles

Reply:
Also I think the logic behind selecting a color would be more appropriate in the viewmodel